tests/foreman/api/test_reporttemplates.py

```diff
@@ -369,22 +379,77 @@ def test_negative_create_report_without_name():
 
 
 @pytest.mark.tier2
-@pytest.mark.stubbed
-def test_positive_applied_errata():
+@pytest.mark.rhel_ver_match(r'^(?!6$)\d+$')
```

Comment:
```suggestion
@pytest.mark.rhel_ver_match('[^6]')
```

Reply:
@Gauravtalreja1 see https://github.com/SatelliteQE/robottelo/pull/11561#discussion_r1211868080.
We have to either come up with some smart solution that would allow us to select all currently supported versions or keep this ugly regex.